frontend/src/App.scss

```diff
@@ -8,6 +8,16 @@
   src: local('Suisse Intl Book'), url(./fonts/SuisseIntl-Book.woff) format('woff');
 }
 
+@font-face {
+  font-family: 'Messer Condensed';
```

Comment:
These are old versions of the files. The current state of master should have the proper references.

Reply:
Merging this in a new PR